spec/spec_helper.rb

```diff
@@ -0,0 +1,42 @@
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'log_decorator'
+require "log4r"
```

Comment:
Why do we need log4r?  Seems like an unnecessary dependency when Ruby has `Logger` built in, and could be used for all of the tests.